src/main/resources/db/migration/V4__Create_template_table.sql

```diff
@@ -0,0 +1,14 @@
+create table template
+(
+    id                  bigserial                not null,
+    name                text                     not null,
+    subject             text,
+    body                text                     not null,
+    creation_timestamp  timestamp with time zone not null,
+    update_timestamp    timestamp with time zone,
+    primary key (id)
+);
+
+alter table message
+    add column template_id bigint references template (id) not null default 1,
```

Comment:
Why the magic 1? IMO, unless we are going to write migrations for existing `subject` and `body` columns in `message` by "manufacturing" corresponding templates, we are better off with no default and re-initializing the database.

Reply:
sure, that works for me